external_parser/unit_tests/test_vw_external_parser.cc

```diff
@@ -247,128 +247,6 @@ BOOST_AUTO_TEST_CASE(ccb_compare_dsjson_with_fb_models) {
                                 buffer_dsjson_model.end());
 }
 
```

Comment:
are these tests removed by mistake?

Reply:
they were already added back if you compare all commits.